main.cpp

```diff
@@ -166,18 +166,32 @@ struct Numeric
     template<typename OtherType>
     Numeric& operator/=(const OtherType& myNumber)
     {
-        if (myNumber == 0.f)
+        if (std::is_same<NumericType, int>::value) // had to change this now, because implicitly converting
+        // types could affect value (and epsilon)
         {
-            if (std::is_same<myType, int>::value) 
+            if (std::is_same<OtherType, int>::value) 
```

Comment:
- [ ] re-read the final instruction for this part and use constexpr.  it'll eliminate the static_cast on 174.

Reply:
Still don't understand what `constexpr` actually does (except that static_cast is not needed anymore).